packages/react-components/component-selector-preview/stories/package.json

```diff
@@ -7,5 +7,8 @@
     "@fluentui/react-storybook-addon-export-to-sandbox": "*",
     "@fluentui/scripts-storybook": "*",
     "@fluentui/eslint-plugin": "*"
+  },
+  "scripts": {
```

Comment:
this is temporary change, will be removed once we will use default target without flag to support legacy format